### PR TITLE
Spanish Latin/es Update for all files

### PR DESCRIPTION
**Pull request by RockLee-BC**
I'm not sure I understand this from github sorry if I bother so much :P

### CODE REVIEW

File: Themes/default/languages/LightPortal/.spanish_es.php [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+	'top' => 'Centro (arriba)',
+	'left' => 'Lado izquierdo',
+	'right' => 'Lado derecho',
+	'bottom' => 'Centro (abajo)',
+	'footer' => 'Pie de página'
+);
+
+$txt['lp_block_areas'] = 'Acciones';
+$txt['lp_block_areas_subtext'] = 'Especifique una o más áreas (separadas por comas) para mostrar el bloque en:<br>
+<ul>
```
Comment (dragomano) [inline Themes/default/languages/LightPortal/.spanish_es.php:125]: Add class="bbc_list" for **ul**.

File: Themes/default/languages/LightPortal/.spanish_latin.php [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <?php
 
 /**
- * Spanish translation by Rock Lee https://www.bombercode.net | Rev. 4.3
+ * .spanish latin language file Rev. 4.3
+ * Spanish translation by Rock Lee https://www.bombercode.net
  *
  * @package Light Portal
+ * @author Bugo https://dragomano.ru/mods/light-portal
```
Comment (dragomano) [inline Themes/default/languages/LightPortal/.spanish_latin.php:8]: I already use a different type of comments in the main language files :)

File: Themes/default/languages/LightPortal/.spanish_es.php (hunk 1 above)
Comment (RockLee-BC) [reply Themes/default/languages/LightPortal/.spanish_es.php]: Apply the change in Latin but not in es, it is already fixed.

File: Themes/default/languages/LightPortal/.spanish_latin.php (hunk 2 above)
Comment (RockLee-BC) [reply Themes/default/languages/LightPortal/.spanish_latin.php]: If I had several doubts like placing it in itself, that's why I put a general one and then changed it.

File: Sources/LightPortal/addons/EasyMarkdownEditor/langs/spanish_latin.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?php
+// .spanish latin language file Rev. 1.1
+// Spanish translation by Rock Lee https://www.bombercode.net
+
+$txt['lp_block_types']['md'] = 'Markdown personalizado'; // Check meaning Markdown
```
[the remark this answers is not in the document]
Comment (RockLee-BC) [reply Sources/LightPortal/addons/EasyMarkdownEditor/langs/spanish_latin.php]: Applied both in Latin as es

**Pull request merged**